lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -1199,6 +1212,72 @@ TfToken MayaDescriptorToToken(const MVertexBufferDescriptor& descriptor)
 
 } // anonymous namespace
 
+class HdVP2Material::TextureLoadingTask
+{
+public:
+    TextureLoadingTask(
+        HdVP2Material*     parent,
+        HdSceneDelegate*   sceneDelegate,
+        const std::string& path,
+        const VtValue&     fallbackValue)
+        : _parent(parent)
+        , _sceneDelegate(sceneDelegate)
+        , _path(path)
+        , _fallbackValue(fallbackValue)
+    {
+    }
+
+    ~TextureLoadingTask() = default;
+
+    void Load()
```

Comment:
I think this function should be renamed to something like EnqueueLoadOnIdle, at least to let the read know it's not doing the work right away.

Reply:
Done.